resources/assets/app.scss

```diff
@@ -2,3 +2,26 @@ $forge-path: "~@dosomething/forge/assets/";
 
 // Import Forge.
 @import "~@dosomething/forge/scss/forge";
+
+.tag {
+  list-style-type: none;
+  clear: both;
+  background: $light-gray;
+  color: $med-gray;
+  font-size: $font-regular;
+  font-weight: $weight-normal;
+  font-family: $font-proxima-nova;
+  text-align: center;
+  border: 0;
+  margin: 10px 10px 0 0;
+
+  &:hover {
+    background: $blue;
+    color: white;
+  }
+
+  &:active {
+    background: $blue;
+    color: white;
+  }
```

Comment:
@chloealee It still shows the gray background? Hmm... can you try forcing a rebuild by killing Webpack and running `npm start` again? Just in case something's cached.